community/CONTRIBUTING.md

```diff
@@ -370,7 +407,7 @@ The OPEA projects use GitHub Action for CI test.
 - End to End Test, the PR must pass all end to end tests.
 
 #### Pull Request Review
-You can add reviewers from [the code owners list](./codeowner.md) to your PR.
+You can add reviewers from [the code owners list](https://github.com/opea-project/docs/blob/8c1df32f043e0a02398cb4a73fd05ed7e2987556/community/codeowner.md) to your PR.
```

Comment:
I don't think people outside of the project have permissions to add reviewers. You may want to strike this section.

Are you intentionally linking to a specific commit?
You may want:
```suggestion
You can add reviewers from [the code owners list](https://github.com/opea-project/docs/blob/main/community/codeowner.md) to your PR.
```

Please also note the code owners list is pretty much empty.